backend/app/assets/javascripts/spree/backend/taxonomy.js.coffee

```diff
@@ -1,9 +1,11 @@
 handle_ajax_error = (XMLHttpRequest, textStatus, errorThrown) ->
-  $.jstree.rollback(last_rollback)
-  $("#ajax_error").show().html("<strong>" + server_error + "</strong><br />" + taxonomy_tree_error)
+  errors = JSON.parse(XMLHttpRequest.responseText).errors
```

Comment:
Need to see what this looks like when the server-side error is unrelated to model validation